app/buyers/views/buyers.py

```diff
@@ -0,0 +1,282 @@
+    )
+    section = content.get_section(section_id)
+    if not section:
+        abort(404)
+
+    return render_template(
+        "buyers/edit_brief_section.html",
+        framework=framework,
+        data=brief,
+        section=section,
+        **dict(buyers.config['BASE_TEMPLATE_DATA'])
+    ), 200
+
+
+@buyers.route(
+    '/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/edit/<section_id>',
+    methods=['POST'])
+def update_brief_submission(framework_slug, lot_slug, brief_id, section_id):
+    framework = data_api_client.get_framework(framework_slug)["frameworks"]
+    if framework["status"] != "live":
+        abort(404)
+
+    try:
+        lot = next(lot for lot in framework['lots'] if lot['slug'] == lot_slug)
+    except StopIteration:
+        abort(404)
+    if not lot['allowsBrief']:
+        abort(404)
+
+    brief = data_api_client.get_brief(brief_id)["briefs"]
+    if not is_brief_associated_with_user(brief, current_user.id) or not brief_can_be_edited(brief):
+        abort(404)
+
+    content = content_loader.get_manifest(framework_slug, 'edit_brief').filter(
+        {'lot': lot['slug']}
+    )
+    section = content.get_section(section_id)
+    if not section:
+        abort(404)
+
+    update_data = section.get_data(request.form)
+
+    try:
+        data_api_client.update_brief(
+            brief_id,
+            update_data,
+            updated_by=current_user.email_address,
+            page_questions=section.get_field_names()
+        )
+    except HTTPError as e:
+        update_data = section.unformat_data(update_data)
+        errors = section.get_error_messages(e.message, lot_slug)
+
+        return render_template(
+            "buyers/edit_brief_section.html",
+            framework=framework,
+            data=update_data,
+            section=section,
+            errors=errors,
+            **dict(buyers.config['BASE_TEMPLATE_DATA'])
+        ), 200
+
+    return redirect(
+        url_for(".view_brief_summary", framework_slug=framework_slug, lot_slug=lot_slug, brief_id=brief_id)
+    )
+
+
+@buyers.route('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>', methods=['GET'])
+def view_brief_summary(framework_slug, lot_slug, brief_id):
+    framework, lot = get_framework_and_lot(framework_slug, lot_slug, data_api_client)
+
+    if framework['status'] != 'live':
+        abort(404)
+    if not lot['allowsBrief']:
+        abort(404)
+
+    brief = data_api_client.get_brief(brief_id)["briefs"]
+    if not is_brief_associated_with_user(brief, current_user.id):
+        abort(404)
+
+    content = content_loader.get_manifest(framework_slug, 'edit_brief').filter(
+        {'lot': lot['slug']}
+    )
+    sections = content.summary(brief)
+    unanswered_required, unanswered_optional = count_unanswered_questions(sections)
+    delete_requested = True if request.args.get('delete_requested') else False
+
+    # TODO: check validation errors(?)
+    validation_errors = None
```

Comment:
I'm not 100% sure - hence the `(?)` - we have it for submitting services in the supplier app and it checks the service against the API for deciding if a service can be submitted or not. We might want the same thing for when we publish briefs, I wasn't sure so left it as a placeholder for now.  

Could just kill it though?  I'm not sure we'll need it, and if we do would be easy to add the check back in?